runbotics/runbotics-orchestrator-ui/src/components/BotProcessRunner.tsx

```diff
@@ -47,12 +49,23 @@ const BotProcessRunner: FC<BotProcessRunnerProps> = ({
     const [modalOpen, setModalOpen] = useState(false);
     const { translate } = useTranslations();
     const hasRunProcessAccess = useFeatureKey([FeatureKey.PROCESS_START]);
-
+    
     const processInstances = useSelector(processInstanceSelector);
+    const { activeJobs } = useSelector(schedulerSelector);
     const { orchestratorProcessInstanceId, processInstance } = processInstances.active;
     const isRunButtonDisabled = started || isSubmitting || !process.system || !process.botCollection;
     const isProcessAttended = process?.isAttended && process?.executionInfo;
-
+    const processName = process?.name;
+    const processId = process?.id;
+    
+    useEffect(() => {
+        dispatch(schedulerActions.getActiveJobs())
```

Comment:
it wont work, there is no assurance that after dispatch even if you would use await that active jobs will be filled with server data. You have to move it to another useEffect and w8 for change in activeJobs variable